Stripe/STPPaymentOptionsInternalViewController.m

```diff
@@ -166,6 +169,31 @@ - (BOOL)isPaymentOptionDetachable:(id<STPPaymentOption>)paymentOption {
     return YES;
 }
 
+- (NSArray<id<STPPaymentOption>> *)cardPaymentOptions {
+    return [self.paymentOptions filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id<STPPaymentOption> _Nullable evaluatedObject, NSDictionary<NSString *,id> * __unused _Nullable bindings) {
+        if ([evaluatedObject isKindOfClass:[STPPaymentMethodParams class]]) {
+            STPPaymentMethodParams *paymentMethodParams = (STPPaymentMethodParams *)evaluatedObject;
+            if (paymentMethodParams.type == STPPaymentMethodTypeFPX) {
+                return NO;
+            }
+        }
+        return YES;
+    }]];
+}
+
+- (NSArray<id<STPPaymentOption>> *)apmPaymentOptions {
+    return [self.paymentOptions filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id<STPPaymentOption> _Nullable evaluatedObject, NSDictionary<NSString *,id> * __unused _Nullable bindings) {
+        if ([evaluatedObject isKindOfClass:[STPPaymentMethodParams class]]) {
+            STPPaymentMethodParams *paymentMethodParams = (STPPaymentMethodParams *)evaluatedObject;
+            if (paymentMethodParams.type == STPPaymentMethodTypeFPX) {
```

Comment:
i think `== STPPaymentMethodTypeFPX` makes sense here since we don't have basic integration support for other apms (would maybe add a comment to that effect)

Reply:
Makes sense.